cmd/security-proxy-setup/main.go

```diff
@@ -64,6 +64,10 @@ func main() {
 		BootTimeout: internal.BootTimeoutDefault,
 	}
 	startup.Bootstrap(params, proxy.Retry, logBeforeInit)
+	if proxy.Configuration == nil {
+		// proxy.LoggingClient wasn't initialized either
+		os.Exit(1)
```

Comment:
Before exit out- add some fmt print out for troubleshooting.

Reply:
Here is what actually happens:
```
../security-proxy-setup --init=true
level=ERROR ts=2019-11-01T18:10:16.2489911Z app=edgex-security-proxy-setup source=logger.go:73 msg="logTarget cannot be blank, using stdout only"
level=ERROR ts=2019-11-01T18:10:16.249481831Z app=edgex-security-proxy-setup source=loader.go:52 msg="could not load configuration file (./res/configuration.toml): open ./res/configuration.toml: no such file or directory"
```